app/main.py

```diff
@@ -1,6 +1,37 @@
+import os
+
+import requests
+
+
+API_KEY = os.getenv("API_KEY")
```

Comment:
Ensure that the `API_KEY` environment variable is set in your environment. If it's not set, `os.getenv("API_KEY")` will return `None`, which will cause issues when making the API request.